app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -27,6 +27,7 @@ class AuthSourceLdap < AuthSource
   include Parameterizable::ByIdName
   include Encryptable
   encrypts :account_password
+  include Taxonomix
```

Comment:
How would you assign taxonomies to an AuthSourceExternal though? There's not even a controller to modify it